folium/folium.py

```diff
@@ -578,6 +578,10 @@ def choropleth(self, geo_path=None, geo_str=None, data_out='data.json',
             keyword argument will enable conversion to GeoJSON.
         reset: boolean, default False
             Remove all current geoJSON layers, start with new layer
+        smooth_factor: float, default None
+            How much to simplify the polyline on each zoom level. More means
+            better performance and smoother look, and less means more accurate
+            representation. Leaflet defaults to 1.0.
```

Comment:
Thanks for the review @ocefpaf. In terms of docs, I agree the repetition isn't great. When you say interpolation, do you mean something like this: https://pypi.python.org/pypi/sphinx-paramlinks/? Alternatively the description could be contained in the `GeoJson` class and `TopoJson`/`choropleth` could both say something like `smooth_factor: See GeoJson class documentation for details`.